website/docs/reference/commands/clone.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "About dbt clone command"
+sidebar_label: "clone"
+id: "clone"
+---
+
+The `dbt clone` command clones selected nodes from the specified state to the target schema(s). This command makes use of the `clone` materialization:
+- If using a data warehouses that support zero-copy cloning of tables, selected models that are tables will be created as clones
+- For views and warehouses that do not support zero-copy cloning, selected models will be created as simple `select * from ...` pointer views
+
+Note: 
+- The state to clone from is based on the location of nodes in the manifest provided to `--state`.
```

Comment:
This will prefer `--defer-state` if specified, otherwise fall back to `--state` (and error if neither is specified)

(I'm adding docs on `--defer-state` and rolling into #3728)